### PR TITLE
🌐 Add Spanish translation of Advanced User Guide - Templates

### PR DESCRIPTION
**Pull request by bezaca**
Hi there :handshake: !
This pull request include the Spanish translation of advanced/templates.md

This pull request continues with the Spanish translation of #1218

### CODE REVIEW

File: docs/es/docs/advanced/templates.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+* Crea un objeto de tipo `templates` que puedas reutilizar posteriormente.
+* Declara un parámetro como <abbr title="Petición">`Request`</abbr> en la *operación de <abbr title="ruta">path</abbr>* que devolverá una plantilla.
+* Usa el objeto `templates` que creaste , para renderizar y devolver un `TemplateResponse`, utilizando <abbr title="Petición">`request`</abbr> como una de las parejas clave-valor en el "contexto" de Jinja2.
+
+```Python hl_lines="4  11  15-16"
+{!../../../docs_src/templates/tutorial001.py!}
+```
+
+!!! Nota
+    Observa que debes pasar la petición como parte de las parejas clave-valor en el contexto de Jinja2. por lo tanto , debes declararlo también en tus *operaciones de <abbr title="ruta">path</abbr>*.
```
Comment (eavv) [inline docs/es/docs/advanced/templates.md:47]:
```suggestion
    Observa que debes pasar la petición como parte de las parejas clave-valor en el contexto de Jinja2. Por lo tanto, debes declararlo también en tus *operaciones de <abbr title="ruta">path</abbr>*.
```

File: docs/es/docs/advanced/templates.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+!!! tip
+    Declarando `response_class=HTMLResponse` , la documentación de la interfaz gráfica sabrá que la respuesta será un HTML.
```
Comment (eavv) [inline docs/es/docs/advanced/templates.md:50]:
```suggestion
    Declarando `response_class=HTMLResponse`, la documentación de la interfaz gráfica sabrá que la respuesta será un HTML.
```

File: docs/es/docs/advanced/templates.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+Esto mostrará el `id`, tomado del `diccionario` utilizado como "contexto":
+
+```Python
+{"request": request, "id": id}
+```
+
+## Plantillas y archivos estáticos
+
+Además puedes usar `url_for()` dentro de una plantilla, y utilizarlo por ejemplo, con <abbr title="Archivos Estáticos">`StaticFiles`</abbr> que hayas montado.
```
Comment (eavv) [inline docs/es/docs/advanced/templates.md:73]:
```suggestion
Además, puedes usar `url_for()` dentro de una plantilla y utilizarlo, por ejemplo, con <abbr title="Archivos Estáticos">`StaticFiles`</abbr> que hayas montado.
```

File: docs/es/docs/advanced/templates.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+{!../../../docs_src/templates/templates/item.html!}
+```
+
+En este ejemplo, está apuntando a un archivo CSS ubicado en `static/styles.css` que contiene:
+
+```CSS hl_lines="4"
+{!../../../docs_src/templates/static/styles.css!}
+```
+
+Y como estas utilizando `StaticFiles`, ese archivo CSS será proporcionado automáticamente por tu aplicación de **FastAPI** en la URL `/static/styles.css` .
```
Comment (eavv) [inline docs/es/docs/advanced/templates.md:85]:
```suggestion
Y como estas utilizando `StaticFiles`, ese archivo CSS será proporcionado automáticamente por tu aplicación de **FastAPI** en la URL `/static/styles.css`.
```

**Review by eavv — CHANGES_REQUESTED** [review]
Litlle punctuation details

**Comment by tiangolo** [comment]
As this PR had requested changes to be applied but has been inactive for a while, it's now going to be closed. But if there's anyone interested, feel free to create a new PR.

**Pull request closed without merging**